src/blocks/events/event-item/edit.js

```diff
@@ -0,0 +1,144 @@
+				/>
+				<div data-page={ String( attributes.pageNum ) }
+					className={ classnames( className, {
+						'is-empty': isEmpty( attributes ),
+						'page-last-item': attributes.lastItem,
+					}, 'coblocks-custom-event-item' ) }
+				>
+					<div className="wp-block-coblocks-event-item__content">
+						<div className={ classnames( textClasses, 'wp-block-coblocks-event-item__dates' ) } style={ textStyles }>
+							<RichText
+								value={ attributes.eventDay }
+								tagName="p"
+								wrapperClassName="wp-block-coblocks-event-item__day"
+								placeholder={ __( 'Day...' ) }
+								onChange={ eventDay => setAttributes( { eventDay } ) }
+								{ ...richTextAttributes }
+							/>
+							<RichText
+								value={ attributes.eventMonth }
+								tagName="h4"
+								wrapperClassName="wp-block-coblocks-event-item__month"
+								placeholder={ __( 'Month...' ) }
+								onChange={ eventMonth => setAttributes( { eventMonth } ) }
+								{ ...richTextAttributes }
+							/>
+							<RichText
+								value={ attributes.eventDate }
+								tagName="h4"
+								wrapperClassName="wp-block-coblocks-event-item__date"
+								placeholder={ __( 'Date...' ) }
+								onChange={ eventDate => setAttributes( { eventDate } ) }
+								{ ...richTextAttributes }
+							/>
+						</div>
+						<div className={ classnames( textClasses, 'wp-block-coblocks-event-item__heading-wrapper' ) } style={ textStyles }>
+							<RichText
+								value={ attributes.title }
+								tagName="h4"
+								wrapperClassName="wp-block-coblocks-event-item__heading"
+								placeholder={ __( 'Write event title...' ) }
+								onChange={ title => setAttributes( { title } ) }
+								{ ...richTextAttributes }
+							/>
+							<RichText
+								value={ attributes.description }
+								tagName="p"
+								wrapperClassName="wp-block-coblocks-event-item__description"
+								placeholder={ __( 'Write event description...' ) }
+								onChange={ description => setAttributes( { description } ) }
+								{ ...richTextAttributes }
+							/>
+						</div>
+						<div className={ classnames( textClasses, 'wp-block-coblocks-event-item__time-and-location' ) } style={ textStyles }>
+							<RichText
+								value={ attributes.eventTime }
+								tagName="h5"
+								wrapperClassName="wp-block-coblocks-event-item__time"
+								placeholder={ __( '10:00pm - 1:00am' ) }
+								onChange={ eventTime => setAttributes( { eventTime } ) }
+								{ ...richTextAttributes }
+							/>
+							<RichText
+								value={ attributes.eventLocation }
+								tagName="p"
+								wrapperClassName="wp-block-coblocks-event-item__location"
+								placeholder={ __( 'Write location...' ) }
+								onChange={ eventLocation => setAttributes( { eventLocation } ) }
+								{ ...richTextAttributes }
+							/>
+						</div>
+					</div>
+					{ attributes.lastItem &&
+					<div className="event-item-last" style={ textStyles }>
+						<div><span>PAGE BREAK</span></div>
```

Comment:
Is this an intentional 'PAGE BREAK' string?

Reply:
Yes, that is only visible in editor.